event_routing_backends/settings/common.py

```diff
@@ -173,26 +173,34 @@ def plugin_settings(settings):
         'edx.course.grade.now_failed'
     ]
 
-    settings.EVENT_BUS_TRACKING_LOGS = set(allowed_xapi_events + allowed_caliper_events)
+    allowed_events = set(allowed_xapi_events + allowed_caliper_events)
+
+    settings.EVENT_BUS_TRACKING_LOGS = allowed_events
 
     settings.EVENT_TRACKING_BACKENDS.update({
-        'xapi': {
+        'events': {
```

Comment:
I think instead of `events` we should have `event_transformer` or `event_routing`.